client/android/app/build.gradle

```diff
@@ -84,7 +84,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion '25.0.0'
```

Comment:
The [react-native docs](https://facebook.github.io/react-native/docs/getting-started.html#3-install-the-android-6-0-marshmallow-sdk) seem to be pretty specific about requiring Android SDK Build-Tools 23.0.1, is there a reason this was upgraded?